Repairs/app/src/main/java/com/rooio/repairs/Dashboard.kt

```diff
@@ -22,8 +24,19 @@ class Dashboard : NavigationBar() {
     private var time: TextView? = null
     private var address: TextView? = null
     private var image: ImageView? = null
+    private var pendingNum: TextView? = null
```

Comment:
All variable definitions should be defined this way:
private lateinit var varName: varType